examples/lighting-app/efr32/BUILD.gn

```diff
@@ -125,6 +131,8 @@ efr32_executable("lighting_app") {
       "$dir_pw_stream",
       "$dir_pw_sys_io",
```

Comment:
This is missing a dependency on pw_stream:sys_io_stream:

```
gen --check out/with_pw_rpc --args='import("//with_pw_rpc.gni")'
ninja -C out/with_pw_rpc
ERROR at //third_party/connectedhomeip/examples/common/pigweed/RpcService.cpp:30:11: Include not allowed.
#include "pw_stream/sys_io_stream.h"
          ^------------------------
It is not in any dependency of
  //:lighting_app.executable
The include file is in the target(s):
  //third_party/connectedhomeip/third_party/pigweed/repo/pw_stream:sys_io_stream
which should somehow be reachable.
___________________
ERROR at //src/Rpc.cpp:32:11: Include not allowed.
#include "pw_stream/sys_io_stream.h"
          ^------------------------
It is not in any dependency of
  //:lighting_app.executable
The include file is in the target(s):
  //third_party/connectedhomeip/third_party/pigweed/repo/pw_stream:sys_io_stream
which should somehow be reachable.
___________________
ERROR at //third_party/connectedhomeip/examples/platform/efr32/PigweedLogger.cpp:31:11: Include not allowed.
#include <pw_stream/sys_io_stream.h>
          ^------------------------
It is not in any dependency of
  //:lighting_app.executable
The include file is in the target(s):
  //third_party/connectedhomeip/third_party/pigweed/repo/pw_stream:sys_io_stream
```

Reply:
Oh cool, never used gn check before. Thanks, fixed it.